joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -1054,4 +1054,30 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
     actualParameter.name shouldBe "bar"
     implicitParameter.name shouldBe "this"
   }
+
+  "have correct structure when a association is passed as an argument with parantheses" in {
+    val cpg = code("""foo(bar:)""".stripMargin)
+
+    cpg.argument.size shouldBe 2
+    cpg.argument.l(0).code shouldBe "bar:"
+    cpg.argument.l(1).code shouldBe "bar"
+
+    cpg.call.size shouldBe 2
+    val List(callNode, operatorNode) = cpg.call.l
+    callNode.name shouldBe "foo"
+    operatorNode.name shouldBe "<operator>.activeRecordAssociation"
```

Comment:
Out of this PR's responsibility, but might as well take advantage of a sample at hands.

I think there shouldn't be an operator here. The sample `foo(bar:)` should mean the same as `foo(bar: bar)`, in which case there wouldn't be an operator, but rather a call to `bar`.

Do you have a different opinion?